doorstop/core/publisher.py

```diff
@@ -524,7 +524,7 @@ def _lines_html(obj, linkify=False, extensions=EXTENSIONS,
                                                      '..', 'views'))
             if 'baseurl' not in bottle.SimpleTemplate.defaults:
                 bottle.SimpleTemplate.defaults['baseurl'] = ''
-            html = bottle_template(template, body=body, toc=toc_html, parent=obj.parent)
+            html = bottle_template(template, body=body, toc=toc_html, parent=obj.parent, obj=obj)
```

Comment:
Maybe `obj=` should be `document=` and `bottle_template()` should be rewritten to use `document.parent`?

Reply:
I refactored it to `document=`
I didn't remove `parent=obj.parent` for backwards compatibility.